Makefile.am

```diff
@@ -258,16 +257,9 @@ CPP_LINT_FILES = $(shell find . \( -name "*.h" -or -name "*.cpp" \) -and ! \( \
         -wholename "./tools/ola_trigger/config.tab.*" -or \
         -wholename "./tools/ola_trigger/lex.yy.cpp" \) | xargs)
 .PHONY : cpplint
-if FOUND_CPPLINT
 cpplint: Makefile.am
-	cpplint.py --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
+if FOUND_CPPLINT
+	cpplint --repository=. --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
```

Comment:
Would --root work if repository is still needed, which would be backwards compatible with the old tool too?

Reply:
```
root=subdir
      The root directory used for deriving header guard CPP variable.
      This directory is relative to the top level directory of the repository
      which by default is determined by searching for a directory that contains
      .git, .hg, or .svn but can also be controlled with the --repository flag.
      If the specified directory does not exist, this flag is ignored.
```
It seems that because I resolved the issue with the missing `.git` folder I can just take out this argument entirely? I'll try that.

Reply:
Yup, that did the trick! Fixed!